### ISSUE TITLE
Request additional sources

### ISSUE THREAD

**Issue by DRSDavidSoft** [mitchellkrogza/Ultimate.Hosts.Blacklist#529]
Hi there, would it be possible to add the following sources as repositories to https://github.com/Ultimate-Hosts-Blacklist?

### dbl.oisd.nl [related issue](https://github.com/dead-hosts/dev-center/issues/5)
* `https://dbl.oisd.nl` (main)
* `https://dblmobile.oisd.nl` (mobile edition)

### MoaAB (i.e. Mother of all adblocking)
* `https://adblock.mahakala.is/` (main)

These lists contain some useful domains to block, but they're bloated with inactive domains, so a CI + PyFunceble sorting and filtering them out will be useful.

Thanks!

**Comment by mitchellkrogza** [comment]
We've just dropped mahakala after it being here for a long time because it just has too many false positives. The other team members can have a look inclusion of oisd.nl

**Comment by spirillen** [comment]
Hi @DRSDavidSoft 

At first eye sight I don't see any differ from other list.

So where is it you think these list add to existing lists?

**Comment by dnmTX** [comment]
@DRSDavidSoft  where is this list coming from? Does it have a repo in GitHub so we can see how well is curated?  
Also we'll wait for @funilrys to test it against all the lists present here to check the number of unique domains and we'll go from there.

**Comment by DRSDavidSoft** [comment]
The MoaAB (`adblock.mahakala.is`) homepage is actually on the XDA forums [here](https://forum.xda-developers.com/showthread.php?t=1916098). I don't believe that they use any sort of CVS (let alone git), but aside from the ridiculous amount of false positives, they're kind of a mostly reputable project. (Besides Energized and 1Hosts projects, which are also XDA-oriented projects).

I agree that the number of false positives for MoaAB list is a lot, but I would still appreciate a CI running PyFunceble on it to detect ACTIVE domains, so that I can cross-reference it with the other sources in my list.

**Comment by dnmTX** [comment]
@DRSDavidSoft  i was asking about `dbl.oisd.nl`. 
`adblock.mahakala.is` was removed and ain't no coming back anytime soon.

**Comment by DRSDavidSoft** [comment]
@dnmTX, oh, I see.

It's worth mentioning that the author has a zero false positive policy and takes this matter very seriously.

> This blocklist contains well over 1 million entries, but still you should not have to whitelist things, it's thát good.
> It's my goal for this list to be as perfect as possible, so you shouldn't find any :D

What they explaine is that they avoid including any sources that may produce a false positive:

> I maintain a separate list (which I call the "skipifcontains"-list. In here I listed domains that should never be blocked. Eg; google.com, facebook.com, amazon.com, microsoft.com, netflix.com, etc) 
> If a downloaded list contains any one of those listed, then I run the list though a regex loop to only capture the domains that make sense to be blocked, but the rest of the list contents will be skipped. 

When I asked him for their sources on Reddit, he linked to only _some_ of them, which is here: https://credits.oisd.nl that's why I'm eager to see what other sources he use.

**Comment by mitchellkrogza** [comment]
That's one massive list of credits :scream_cat:  I can almost guarantee you there is no way a list like that can be properly maintained and not contain false positives. Several of my own projects included there but just a quick look through reveals a number of other lists which have been dropped over the years due to not being maintained and containing to many FP's. There's also a lot of duplication due to the projects included there which are already projects made up of unified lists.

**Comment by DRSDavidSoft** [comment]
@mitchellkrogza This is true, and I agree with you on your points! While that list is extensive (and may contain many redundant entries that serve no purpose to many of the users), it usually contains the bare minimum FPs, and generally doesn't block anything other than what they have advertised to block.

I say this because I maintain a list of whitelist sources (in addition to my list of blacklist sources :3) and whenever I add a new blacklist to my list, I always cross-reference them for any potential FPs, to see if it'll break anything useful in the future. So far, MoaAB seems to contains the most FPs while the oisd list contains the least FPs between any of the aggregated lists that I use.

With that being said, I respect the decision to omit including oisd and/or MoaAB in your repo; it was just a request to see if they'd be accepted here 😄  as I already use the other, more useful lists from your repos.

As for duplicates issues, BTW, I believe it's the user's responsibility to check for and remove any duplicates that might be there, when combining lists from different sources. I know I do, and there are many tools available to do just that for users.

**Comment by dnmTX** [comment]
RIGHT ON @mitchellkrogza 🚀

**Comment by funilrys** [comment]
Well, If `so a CI + PyFunceble sorting and filtering them out will be useful` is the only requirement. It's a good idea I didn't delete it yet from @dead-hosts.

I pinged @mitchellkrogza out there (@dead-hosts ==> dev-center) for the case that he needed help for the setup and configuration of PyFunceble in his very own machine.

In between, we just tested 100000 domains in about 2 hours with my (dev version) changes in the last hours. So let's see what happens with that huge list inside @dead-hosts.

I think everybody will agree, with my decision to close this. 
Let me know if some clarification or others are needed.

Cheers,
Nissar
